.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'Type: Bug'
+assignees: ''
+---
+
+## Bug Description
+
+[Provide a clear and concise description of the bug.]
+
+## Steps to Reproduce
+
+[Describe the steps to reproduce the bug in a detailed manner.]
+
+1. Step 1
+2. Step 2
+3. ...
+
+## Expected Behavior
+
+[Describe what you expected to happen.]
+
+## Current Behavior
+
+[Describe what is currently happening instead.]
+
+## Command Used
+
+[Include the exact command(s) you used when encountering the bug.]
+
+## Screenshots or Output
```

Comment:
I think this field might be removed to make the template shorter (and less scary). People will mention the command in "Current Behavior"